tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -47,7 +47,11 @@ sub run {
     $variables{HANA_ACCOUNT} = get_required_var('QESAPDEPLOY_HANA_ACCOUNT');
     $variables{HANA_CONTAINER} = get_required_var('QESAPDEPLOY_HANA_CONTAINER');
     if (get_var('QESAPDEPLOY_HANA_TOKEN')) {
-        $variables{HANA_TOKEN} = get_required_var('QESAPDEPLOY_HANA_TOKEN');
+        $variables{HANA_TOKEN} = qesap_az_create_sas_token(storage => get_required_var('QESAPDEPLOY_HANA_ACCOUNT'),
```

Comment:
We need to try this in AWS to confirm it works, since we're using the blob in Azure, it'll be an AWS test also running `az` commands.

Reply:
Good point, indeed it does not work. http://openqaworker15.qa.suse.cz/tests/257070#step/configure/66

Reply:
solved in last update